plugins/org.eclipse.papyrus.uml.diagram.sequence.contribution/plugin.xml

```diff
@@ -117,6 +117,12 @@
           name="Lightweight Sequence Diagram"
           category="org.eclipse.papyrus.infra.gmfdiag.preferences.diagrams">
     </page>
+    <page
```

Comment:
I can understand that the existing page (Papyrus standard diagram settings) is already overfull, so we need a separate page.  But perhaps it should be nested under the **Lightweight Sequence Diagram** page and then can just be named **Editing** (with `lightweight` and `sequence` keywords for quick search, of course).